superset-frontend/src/components/Chart/DrillBy/DrillByMenuItems.tsx

```diff
@@ -311,6 +310,7 @@ export const DrillByMenuItems = ({
                 margin: ${theme.gridUnit * 2}px ${theme.gridUnit * 3}px;
                 box-shadow: none;
               `}
+              value={searchInput}
```

Comment:
Same for this. It seems unrelated to the upgrade? What is this for?

Reply:
This is said binding of the state.

Reply:
I think this approach does not work. Watch the example video. I cannot even type right, it will lag really bad. 


https://github.com/user-attachments/assets/c94d51e2-36fb-4b92-8677-303a83accdac